spec/requests/memberships_controller_spec.rb

```diff
@@ -319,4 +319,17 @@
       end
     end
   end
+
+  describe '#reject' do
+    include_context 'current_user is the group leader'
+    context 'when membership has DEFAULT post' do
+      let(:membership) { create(:membership, :newbie) }
+      it 'rejects the membership' do
+        delete "/groups/#{membership.group.id}/memberships/#{membership.id}/reject", xhr: true
+
+        expect(response).to have_http_status(:ok)
+        expect(Membership.exists?(id: membership.id)).to be false
```

Comment:
What do you think about?
```suggestion
        expect(Membership).not_to exist(id: membership.id)
```